src/go/rdctl/pkg/plist/plist_test.go

```diff
@@ -101,8 +101,7 @@ func TestJsonToPlistFormat(t *testing.T) {
   },
   "virtualMachine": {
     "memoryInGB": 4,
-    "numberCPUs": 2,
-    "hostResolver": true
```

Comment:
I'm not 100% sure that's relevant (the test file doesn't really have much to do with the real thing does it?), but there's no harm in removing it here. 👍